lib/drivers/bayerContourNext.js

```diff
@@ -324,16 +324,18 @@ module.exports = function (config) {
                   debug(err);
                   return whilstCb(err);
                 } else {
-                  robj.timestamp = parseInt(r[4]);
-                  robj.annotations = getAnnotations(r[3], data);
-                  robj.control = isControl(r[3]);
-                  robj.units = r[2];
-                  if(robj.units === 'mmol/L') {
-                    robj.glucose = parseFloat(r[1]);
-                  } else {
-                    robj.glucose = parseInt(r[1]);
+                  if(r) {
+                    robj.timestamp = parseInt(r[4]);
+                    robj.annotations = getAnnotations(r[3], data);
+                    robj.control = isControl(r[3]);
+                    robj.units = r[2];
+                    if(robj.units === 'mmol/L') {
+                      robj.glucose = parseFloat(r[1]);
+                    } else {
+                      robj.glucose = parseInt(r[1]);
```

Comment:
can the `parse...` functions ever produce errors? ... might not matter here but just wondering